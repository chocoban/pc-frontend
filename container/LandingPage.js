import { Component } from 'react';
import { connect } from 'react-redux';
import { Button } from 'antd';

class SignUpPage extends Component {
  state = { size: 'large', };

  handleSizeChange = e => {
    this.setState({ size: e.target.value });
  };

  render() {
    const { size } = this.state;

    return (
      <div style={{
        minHeight: '100vh',
        backgroundImage: 'url(../public/static/landingPage/what-is-petty-cash.png)',
        backgroundSize: 'cover'
      }}>
        <div style={{ marginLeft: '25vw', paddingTop: '35vw' }}>
          <Button
            type='primary'
            shape='round'
            size={ size }
            style={{ width: '10vw', height: '3vw' }}
          >
            <a href='http://localhost:8080/google'>
              Sign up
            </a>
          </Button>
        </div>
      </div>
    );
  }
}

const mapStateToProps = state => state;

export default connect(mapStateToProps)(SignUpPage);
