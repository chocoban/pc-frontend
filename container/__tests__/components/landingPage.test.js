import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import LandingPage from '../../LandingPage';

const initialState = {};
const mockStore = configureStore();
const store = mockStore(initialState);
describe('Landing page', () => {
  it('should test the landing page', () => {
    const landingPage = shallow(
      <Provider store={ store }>
        <LandingPage />
      </Provider>
    );
    expect(landingPage).toMatchSnapshot();
  });
});
