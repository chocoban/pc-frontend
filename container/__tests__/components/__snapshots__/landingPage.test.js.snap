// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page should test the landing page 1`] = `ShallowWrapper {}`;
